usr/src/uts/common/fs/zfs/spa.c

```diff
@@ -5509,6 +5511,7 @@ spa_vdev_remove(spa_t *spa, uint64_t guid, boolean_t unspare)
 		 */
 		spa_vdev_remove_from_namespace(spa, vd);
 
+		spa_event_notify(spa, vd, ESC_ZFS_VDEV_REMOVE_AUX);
```

Comment:
Log devices are not "aux" devices so if you want a sysevent for log device removal (and eventually data devices) it would make sense to also define ESC_ZFS_VDEV_REMOVE_DEV. It's too bad that we didn't use ESC_ZFS_VDEV_DETACH instead of ESC_ZFS_VDEV_REMOVE.